com.regnosys.rosetta/model/Rosetta.xcore

```diff
@@ -287,20 +287,30 @@ enum RosettaSynonymRef {
  * docReferences
  */
 
-class RosettaDocReference extends RosettaRegulatoryBody {
+class RosettaDocReference {
```

Comment:
Sneaky :-)

Reply:
You think we should deprecate this type? Usages should be migrated to `RegulatoryDocumentReference`

Reply:
I think the naming is confusing, because they are both actually needed. :) A `RosettaDocReference` represents the annotation, while `RegulatoryDocumentReference` represents a reference to a part of a regulatory document, e.g., a body, a list of corpuses and a list of segments. `RosettaDocReference` contains a `RegulatoryDocumentReference`, but also `rationale`s, `provision`, etc.